### ISSUE TITLE
JSON URL returned from ETrade for Lots

### ISSUE THREAD

**Issue by nickjag** [tflanagan/node-e-trade#1]
First off, awesome library, has been immensely helpful, thank you!

To get specific lots, it looks like etrade provides them, accessible through returned URLs in the responses:

E.g. 
`{ lotsDetails: 'https://api.etrade.com/v1/accounts/<some id>/portfolio/<some id>.json' }`

What would be the easiest way to sign this URL and execute it using this library?

And/or know of getting the lots for specific executed orders an easier way? I've been through the order, instrument and transaction details, but never found any info on lots.

Thanks!

**Comment by tflanagan** [comment]
@nickjag 

I can't seem to find any documentation on `lotsDetails` or `lots` either.

I'll add the method to the library at some point today.

`getBasicRequest`, `signRequest`, and `request` are all private methods, you wouldn't be able to access them directly (well... js) but in essence:

```js
async viewLotsDetails({ accountIdKey, positionId }: ViewLotsDetails){
	const requestOptions = this.getBasicRequest({
		url: `accounts/${accountIdKey}/portfolio/${positionId}.json`
	});

	this.signRequest(requestOptions);

	return this.request<any>(requestOptions); // TODO: find documentation for lots for more specific typing
}
```

**Comment by tflanagan** [comment]
Added in f124aa3 and is available in v0.2.0

**Comment by nickjag** [comment]
Awesome @tflanagan I'll pull and give it a go tomorrow 👍
